test/actions/katello/repository_test.rb

```diff
@@ -208,21 +208,23 @@ def setup
     end
 
     it 'plans ACS product removal when removing the last URL for a product' do
-      ::Katello::SmartProxyAlternateContentSource.create!(alternate_content_source_id: simplified_acs.id, smart_proxy_id: proxy.id, repository_id: repository.id)
+      ::Katello::SmartProxyAlternateContentSource.create!(alternate_content_source_id: simplified_acs.id, smart_proxy_id: proxy.id, repository_id: published_rhel7_repository.id)
       action = create_action action_class
-      action.stubs(:action_subject).with(repository)
+      action.stubs(:action_subject).with(published_rhel7_repository)
 
       # manually remove the URLs from all repos in product except repository
-      repository.product.repositories.each do |repo|
-        repo.root.update!(url: nil) unless repo.id == repository.id
+      testing_repo = repository.product.repositories[4]
```

Comment:
@qcjames53 I'm not sure if the repository at index [4] will consistently be the same one, we'd probably need to look up by name instead.

Reply:
Actually.... it looks like this was applied to the wrong test? The failing test is `plans ACS product removal when removing the deleting the last repo with URL`. I'll see about updating that other test to see if it helps.